tests/src/Kernel/fixtures/markup_rendering_patterns/content_banner.yml

```diff
@@ -28,8 +31,9 @@ content_banner_white_background_with_image_with_badges_with_meta_with_links:
       'div.bcl-content-banner': 1
       'div.bcl-content-banner.bg-white': 1
       'img': 1
+      'div.bcl-content-banner > div.container > div.action-bar': 1
```

Comment:
we should also check for the button you added

Reply:
Changed it to anchor tag. I don't understand why the phpunit is not rendering the button. It should not be a problem with the template or the new parameter.